py/picca/io.py

```diff
@@ -274,8 +276,10 @@ def read_from_mock_1D(in_dir,thid,ra,dec,zqso,plate,mjd,fid,order,mode,log=None)
     except IOError:
         log.write("error reading {}\n".format(fin))
 
+    ih = 0
     for t,r,d,z,p,m,f in zip(thid,ra,dec,zqso,plate,mjd,fid):
-        h = hdu[t]
+        h = hdu[index[0][ih]+1]
```

Comment:
@solenechabanier why can't you use `t` as an index here? Did something change in the simulated data? The introduction of this `index` variable seems a bit strange...

Reply:
We introduced it because we needed two thingID, one for the position of the spectrum in the fits file and the other to match with the DLAs from the catalog

Reply:
Why are they different?